package.json

```diff
@@ -9,6 +9,7 @@
     "gh-pages": "^2.2.0",
     "prop-types": "^15.7.2",
     "react": "^16.13.0",
+    "react-calendar": "^3.0.1",
```

Comment:
@devin-dumais do we need this file?

Reply:
Yes because you added the `react-calendar` dependency. `package.json` tells `yarn` which outside packages to install that this project depends on to function correctly.